projects/sdk/src/lib/silo.ts

```diff
@@ -456,18 +483,25 @@ export class Silo {
    */
   async getPlantableSeeds(_account?: string) {
     const account = await Silo.sdk.getAccount(_account);
-    // TODO: this is wrong
-    return Silo.sdk.contracts.beanstalk.balanceOfEarnedSeeds(account).then((v) => Silo.sdk.tokens.SEEDS.fromBlockchain(v));
+    throw new Error("Not implemented");
```

Comment:
I guess this is more like "Plantable BDV" now? Since I have some BDV in Beans that I can plant which don't earn Stalk until I plant. For now, throwing an error here and can ship a separate PR to discuss.

Reply:
not sure why this wouldn't be Plantable Seeds

Reply:
Seeds no longer exist